### ISSUE TITLE
Sandbox service is down all day

### ISSUE THREAD

**Issue by alexei-serjantov** [hmrc/self-assessment-api#792]
* **I'm submitting a ...**
  - [x] Bug - API
API call self-assessment/ni/RY785126A/savings-accounts

* **What is the current behaviour?**
statusCode=500
 {"code":"INTERNAL_SERVER_ERROR","message":"An internal server error occurred"}

* **What is the expected behaviour?**
statusCode=200

{"savingsAccounts":[{"id":"SAVIZE0hQNiB4r1","accountName":"Main account name"},{"id":"SAV38WpAhiodTxW","accountName":"Shares savings account"}]}

**Comment by MarkAKelly** [comment]
Hi @alexei-serjantov,

We've located the issue, and its a bit of a strange one.  The stateless database that we have serving the API with the test scenarios you use was returning duplicates of the data, which was resulting in 500s at your end.  This was likely caused by a platform issue with the settings of the DB provider which we are looking into.

That being said, we've cleared down the Sandbox data and it all seems to be working as expected.  

Sorry for the inconvenience!
